src/osdatahub/NGDAPI/crs.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+EPSG = {
+    "epsg:4326": "http://www.opengis.net/def/crs/EPSG/0/4326",
+    "epsg:27700": "http://www.opengis.net/def/crs/EPSG/0/27700",
+    "epsg:3857": "http://www.opengis.net/def/crs/EPSG/0/3857",
+    "epsg:7405": "http://www.opengis.net/def/crs/EPSG/0/7405",
+    "crs84": "http://www.opengis.net/def/crs/OGC/1.3/CRS84"
+}
+
+
+# TODO: add url possibility
+def get_crs(crs: str = None, epsg: int = None, valid_crs: Union[list[str], tuple] = EPSG.keys()) -> str:
+    if crs and epsg:
+        raise ValueError("Must pass either crs or epsg, but not both")
+    if epsg:
+        crs = f"epsg:{epsg}"
+
+    if crs in EPSG and crs in valid_crs:
```

Comment:
What's the purpose of having both checks here? Could you just check the valid crs (assuming you've got a different subset per product)?

Reply:
Could you clarify this more?

Reply:
Actually had a think about it and it makes as is - the EPSG check makes sure that the valid_crs are themselves valid?